build.gradle

```diff
@@ -113,22 +113,22 @@ task runCorrelation(dependsOn: ['classes'], type: JavaExec) {
 }
 
 task exportCatalogue(dependsOn: ['classes'], type: JavaExec) {
-    maxHeapSize = "4096m"
+    maxHeapSize = "8192m"
     main = 'uk.org.tombolo.CatalogueExportRunner'
     classpath = sourceSets.main.runtimeClasspath
     jvmArgs '-disableassertions'
 
-    doLast {
+    doFirst {
         checkEnv()
         systemProperty("databaseURI", databaseURI)
         systemProperty("databaseUsername", databaseUsername)
         systemProperty("databasePassword", databasePassword)
         systemProperty("environment", "export")
 
-        def argumentsSet2 = project.hasProperty('outputFile')
+        def argumentsSet2 = project.hasProperty('output')
         if (!argumentsSet2) throw new GradleException('Required properties not specified. See the README.')
```

Comment:
Feel free to change the name of the variable here to something like: `argumentSet`.